### ISSUE TITLE
New database.json (09-09) missing content?

### ISSUE THREAD

**Issue by FritzHeiden** [cta-wave/Test-Content#18]
After redeploying the dpctf test runner we noticed tests are failing due to missing content. I investigated the problem and noticed that the [latest database.json](https://github.com/cta-wave/Test-Content/blob/4447bc47ca97f284ef5c3cdd0353e96408506ec0/database.json) is missing content in comparison to the [old one](https://github.com/cta-wave/Test-Content/blob/1197c7d793fefeeb2d3ecff2815c36f7e9e97581/database.json). The framerate set 15 30 60 only contains a single encrypted test. Non-encrypted content t1-t30 are missing, thus causing [issues](https://github.com/cta-wave/dpctf-deploy/issues/25#issuecomment-931059196) with the current tests in dpctf.

Is this intended or a bug?

**Comment by rbouqueau** [comment]
Intended, see https://github.com/cta-wave/Test-Content-Generation/issues/39

**Comment by FritzHeiden** [comment]
> Intended, see [cta-wave/Test-Content-Generation#39](https://github.com/cta-wave/Test-Content-Generation/issues/39)

This issue is about what encrypted content is needed. However, I am talking about the unencrypted content that was present in the old database.json and is now missing in the new one.

E.g. https://dash.akamaized.net/WAVE/vectors/2021-07-07/avc_sets/15_30_60/t1/stream.mpd was present but 
https://dash.akamaized.net/WAVE/vectors/2021-09-09/avc_sets/15_30_60/t1/stream.mpd is missing.

I can see the content is available [here](https://dash.akamaized.net/WAVE/vectors/2021-09-09/avc_sets/15_30_60/) but database.json does not contain any, except the t1-cenc.

**Comment by rbouqueau** [comment]
> I can see

Unfortunately the server we use rely heavily on caching. Therefore HTTP browsing is not relevant e.g. the server returns inconsistent listing across users. We have to live with it.

The database is certainly more reliable. It is generated along with the content.

> E.g. https://dash.akamaized.net/WAVE/vectors/2021-07-07/avc_sets/15_30_60/t1/stream.mpd was present but
https://dash.akamaized.net/WAVE/vectors/2021-09-09/avc_sets/15_30_60/t1/stream.mpd is missing.

I can download both.

> This issue is about what encrypted content is needed.

Please re-open cta-wave/Test-Content-Generation#39 (or a new issue) if the issue is about the content that's generated. If you think the issue is in the database.json or in the front-end then this is the right place to discuss it.

I'm not sure I answered your question. Please let us know what can be done to fix your issue.

**Comment by louaybassbouss** [comment]
@rbouqueau after more investigation we found an issue with database.json. you use the "CENC" key twice and this cause the issue. Can you please fix it and use keys only once in the JSON structure.

**Comment by rbouqueau** [comment]
My bad. Should be fixed now. Please close if solved.

**Comment by rbouqueau** [comment]
@FritzHeiden  Can we close?

**Comment by FritzHeiden** [comment]
Yes everything is fine now.
